app/_locales/en/messages.json

```diff
@@ -1787,7 +1787,7 @@
     "message": "Connect your Ledger hardware wallet to interact with dApps and make transfers to other connected wallets."
   },
   "metamaskImportSubText": {
-    "message": "Note: If importing from a Metamask wallet, enter 12 words instead of 24."
+    "message": "Note: If importing from a Brave wallet created before February 2021, your secret phrase is 24 words instead of 12."
```

Comment:
note: `February 2021` should be changed to a release number once we know what release this is landing in